scripts/system/html/css/materialAssistant.css

```
@@ -0,0 +1,215 @@
+/*
+//  materialAssistant.css
+//
+//  Created by Alezia Kurdis on May 17th, 2022.
+//  Copyright 2022 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+*/
+
+#uiMaterialAssistant {
+    position: fixed;
+    display: none;
+    width: 100%;
+    top: 0px;
+    left: 0px;
+    right: 0px;
+    bottom: 0px;
+    border: 0px;
+    padding: 0px;
+    background-color: #404040;
+    z-index: 2;
+    border-collapse: collapse;
+    overflow-y: scroll;
+}
+
+#uiMaterialAssistant-headerContainer {
+    width: 98%;
+    text-align: right;
+    padding: 6px;
+    display: flex;
+}
+
+#uiMaterialAssistant-closeButton {
+    font-family: Raleway-Bold;
+    font-size: 18px;
+    text-align: center;
+    border: 0px;
+    padding: 6px;
+    color: #FFFFFF;
+}
+
+#uiMaterialAssistant-sidewalk {
+    width: 100%;
+    background-image: url("img/sidewalk.png"); 
+    background-repeat: repeat;
+}
+
+#uiMaterialAssistant-formContainer {
+    background-color: #404040;
+    width: 90%;
+    padding: 2px;
+}
+
+div.uiMaterialAssistant-group{
+    background-color: #2E2E2E;
+    border: 2px solid #aaaaaa;
+    padding: 5px;
+    margin: 0px 0px 6px 0px;
+    font-family: Raleway-SemiBold;  
+    text-decoration: none; 
+    font-size: 13px; 
+    color:#ffffff;   
```

```suggestion
    color: #ffffff;
```
And similar elsewhere.

---

Fixed.